### PR TITLE
Fixes for CircleCI and Packet

### PR DESCRIPTION
**Pull request by nickolaev**
## Description
An attempt to optimize (and speed up?) the CI Packet deployment.

## Motivation and Context
Terraform on Packet was saying there are deprecated options. Fixing this, went also through the state of the circleci.yaml and made it use the latest golang image. It is the same approach as when we build the files, so better consistency.
Packet instances are explicitly named to t1.small.x86 and the k8s installation script is simplified.

## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -42,40 +42,9 @@ jobs:
     environment:
       GO111MODULE: "on"
```
Comment (edwarnicke) [inline .circleci/config.yml]: Why cut out all the sanity checks?

**Comment by edwarnicke** [comment]
Switching from Ubuntu 16.04 to Ubuntu 18.04 takes us out fo the 'fast deploy' scenario for Packet... which means an extra 2+ minutes on the packet-deploy job and thus the overall CI time...

File: .circleci/config.yml (hunk 1 above)
Comment (nickolaev) [reply .circleci/config.yml]: @edwarnicke there are 2 separate jobs `sanity-check` and `build`. Apparently, the latter is never referenced and looks like a stale historical piece just laying around. I propose to remove it.

**Comment by nickolaev** [comment]
OK so 18.04 is reverted.

**Pull request merged**